app/models/result.rb

```diff
@@ -12,4 +12,10 @@ def initialize(title, year, url, type)
     @url = url
     @type = type
   end
+
+  # View-type method for returning a truncated list of authors.
+  def truncated_authors
+    return authors if authors.length <= ENV['MAX_AUTHORS'].to_i
+    authors[0..ENV['MAX_AUTHORS'].to_i - 1] << 'et al'
```

Comment:
A more ruby way to write this is:
`authors[0...ENV['MAX_AUTHORS'].to_i] << 'et al'`

See: http://ruby-doc.org/core-2.3.1/Range.html